contracts/lib/SafeMath.sol

```diff
@@ -0,0 +1,159 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.6.0;
+
+/**
+ * @dev Wrappers over Solidity's arithmetic operations with added overflow
+ * checks.
+ *
+ * Arithmetic operations in Solidity wrap on overflow. This can easily result
+ * in bugs, because programmers usually assume that an overflow raises an
+ * error, which is the standard behavior in high level programming languages.
+ * `SafeMath` restores this intuition by reverting the transaction when an
+ * operation overflows.
+ *
+ * Using this library instead of the unchecked operations eliminates an entire
+ * class of bugs, so it's recommended to use it always.
+ */
+library SafeMath {
```

Comment:
You can directly import it from openzeppelin

```solidity
import { SafeMath } from "@openzeppelin/contracts/math/SafeMath.sol";
```

Also, I'll add the flattener script in this project so that it'd be convenient for you to get the contracts to remix to deploy comfortably.